### ISSUE TITLE
Support globbing in excluded_entities

### ISSUE THREAD

**Issue by postlund** [home-assistant/ui-schema#233]
I would like to keep my "Unused entities" as clean as possible, but it is know cluttered with various automations, scripts and other things that i will never add to the UI. So I would like to ignore them. Since especially scripts and automations will vary over time depending on what I do, it would be convenient if I could remove them with a wildcard (or at least exclude domains).

**Comment by balloob** [comment]
I wonder if we should just group it by domain instead, that way you can just gloss over the domain cards that you don't care about.

**Comment by postlund** [comment]
Yeah, that's one way. Or extend Lovelace so that entities can be hidden directly from the UI (press an "X" next to it or something). That way it's easy to remove entities, so there would be no real problems when adding for instance new automations or scripts. It would probably make more sense since you would have to see all entities at least once (they would otherwise be automatically hidden).

**Comment by postlund** [comment]
Can't it just update `excluded_entities` in the Lovelace config, like when adding or removing cards? Doesn't have to be a separate thing.

**Comment by thomasloven** [comment]
How often are you expecting to look at the Unused entities list?
